packages/commonwealth/client/scripts/views/pages/discussions/DiscussionsPage.tsx

```diff
@@ -94,13 +107,19 @@ const DiscussionsPage = ({ topicName }: DiscussionsPageProps) => {
             `${thread.identifier}-${slugify(thread.title)}`
           );
 
-          const canReact =
-            hasJoinedCommunity && !thread.lockedAt && !thread.archivedAt;
+          const disabledActionsTooltipText = (() => {
```

Comment:
I'm seeing this syntax in numerous places, let's see if we can't make this follow the DRY principle more closely.

Reply:
done